Source/Scene/Cesium3DTileset.js

```diff
@@ -929,7 +929,7 @@ function Cesium3DTileset(options) {
    * @type {Metadata3DTilesExtension}
    * @private
    */
-  this.metadata = undefined;
+  this._metadataExtension = undefined;
```

Comment:
Make sure to update `ImplicitSubtree` as well: https://github.com/CesiumGS/cesium/blob/6d40ab1ba06f548235c2205e088d65c555421567/Source/Scene/ImplicitSubtree.js#L624

Reply:
indeed, though that's an integration detail for #9445. I'll put a note in that PR.